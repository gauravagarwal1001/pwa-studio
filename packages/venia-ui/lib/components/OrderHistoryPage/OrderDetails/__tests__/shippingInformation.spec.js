import React from 'react';
import { createTestInstance } from '@magento/peregrine';

import ShippingInformation from '../shippingInformation';

const defaultProps = {
    data: {
        city: 'Austin',
        country_code: 'US',
        firstname: 'Gooseton',
        lastname: 'Jr',
        postcode: '78759',
        region_id: 'TX',
        street: 'Goose Dr',
        telephone: '9123456789'
    }
};

test('should render properly', () => {
    const tree = createTestInstance(<ShippingInformation {...defaultProps} />);

    expect(tree.toJSON()).toMatchSnapshot();
});
