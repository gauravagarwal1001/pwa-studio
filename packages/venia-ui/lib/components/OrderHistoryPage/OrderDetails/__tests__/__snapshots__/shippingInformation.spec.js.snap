// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<div>
  <div>
    Shipping Information
  </div>
  <div>
    <span>
      Gooseton
    </span>
    <span>
      Jr
    </span>
  </div>
  <div>
    Goose Dr
  </div>
  <div>
    Austin, TX, 78759
  </div>
  <div>
    US
  </div>
</div>
`;
